grafana/README.md

```diff
@@ -0,0 +1,81 @@
+Grafana Dashboard
+=================
+
+This folder contains docker compose configuration to configure
+grafana/influxdb/telegraf to query (every 10 seconds) your AQI
+monitors, and to chart the AQI over time.
+
+Setup
+-----
+
+1. Edit `docker-compose.yaml` and change the IP address (192.168.0.2) to
+   your server IP address (the machine where the docker environment will run)
+
+2. Edit `telegraf/telegraf.d/inputs.homebrew_aqi.conf` to point at your
```

Comment:
Could this be changed to used [an environment variable](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#environment-variables) passthrough docker compose via a `.env` file?

Reply:
Interesting - I wonder if that works for lists... will try